### DISCUSSION TITLE
Formular support?

### DISCUSSION THREAD

**Discussion by XMethues** [toeverything/blocksuite#6554] [Ideas]
Does notion like formula will be supported in the future？

**Comment by doouding** [comment]
I guess you're looking for LaTeX capabilities; we already have plans for that, so stay tuned.

**Reply by XMethues** [reply]
Not LaTeX. I mean formulars in data table.

**Comment by balupton** [comment]
Yeah, I'm really after a Coda.io alternative, in which you can use `/` to create formulas, variables, and queries onto tables, apis, plugins, etc.

Created https://github.com/toeverything/blocksuite/discussions/9176 for all the coda.io features.
